app/models/loan.rb

```diff
@@ -92,26 +92,21 @@ def renewable?
 
   # Can a member renew this loan themselves without approval?
   def member_renewable?
-    renewable? && within_borrow_policy_duration? && item.borrow_policy.member_renewable? && ended_at.nil?
+    renewable? && item.borrow_policy.member_renewable? && ended_at.nil?
   end
 
   # Can a member request this loan be renewed?
   def member_renewal_requestable?
-    renewable? && within_borrow_policy_duration? && ended_at.nil? && !any_active_holds? && !renewal_requests.any?
+    renewable? && ended_at.nil? && !any_active_holds? && !renewal_requests.rejected.any?
```

Comment:
Is there any reason for adding the `rejected` scope here? Wouldn't this make it possible for a member to create multiple renewal requests for a single loan at the same time?

Reply:
That's a good call. I'll take out the `rejected` call here.

Reply:
Oh, maybe it should be `requested` so that a member can't request a renewal for something that they already have?